modules/osd-monitoring-configurable-monitoring-components.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="configurable-monitoring-components_{context}"]
+= Configurable monitoring components
+
+This table shows the monitoring components you can configure and the keys used to specify the components in the `cluster-monitoring-config` and `user-workload-monitoring-config` `ConfigMap` objects:
+
+.Configurable monitoring components
+[options="header"]
+|====
```

Comment:
I think we just need to list
Prometheus Operator, Prometheus, Thanos Ruler for `user-workload-monitoring-config config map key`, `cluster-monitoring-config ConfigMap key` is not needed

Reply:
Ok, commented out the items that are not needed.